powerstore/resource_host.go

```diff
@@ -531,64 +508,144 @@ func (r resourceHost) planToServer(plan, state models.Host) *gopowerstore.HostMo
 	}
 
 	hostUpdate = &gopowerstore.HostModify{
-		Description:      &description,
-		HostConnectivity: gopowerstore.HostConnectivityEnum(plan.HostConnectivity.ValueString()),
-		Name:             &name,
-		AddInitiators:    &addInitiators,
 		RemoveInitiators: &removeInitiators,
 	}
 
 	return hostUpdate
 }
 
 // to perform modify operation in update
-func (r resourceHost) modifyOperation(plan, state models.Host) *gopowerstore.HostModify {
-	hostUpdate := &gopowerstore.HostModify{}
+func (r resourceHost) modifyOperation(plan, state models.Host) []gopowerstore.UpdateInitiatorInHost {
+	//hostUpdate := &gopowerstore.HostModify{}
```

Comment:
Pls remove the comment if its not required

Reply:
removed